### ISSUE TITLE
Are we still working on this one?

### ISSUE THREAD

**Issue by xrotwang** [lexibank/hruschkaturkic#3]
Since this dataset is given as data source for a phlorest phylogeny, there should be a release, I think.

**Comment by LinguList** [comment]
Yep, but it is not really trivial, but yes, one can finalize this and release without any concepts (as there are none).

**Comment by LinguList** [comment]
Well, the data was extracted from an etymological dictionary and manually brought into the form of alignments. Theoretically, we can try and trace the root forms for 400+ Turkic roots from the Starostin database, but this is by far not easy, or let's say a rather time consuming task. This would give us etyma, similar to the Siouan DB, but no concepts for extant forms, as they were dropped or never accounted for.

**Comment by LinguList** [comment]
So my approach would be to say that this dataset is useful for testing alignment algorithms, as it comes with alignments, and potentially for some attempts to reconstruct Proto-Turkic forms or sound change processes. But nothing more.

**Comment by xrotwang** [comment]
Ok, I see. So, @SimonGreenhill , I'd propose to just move the raw data to the phlorest phylogeny where it is referenced, rather than pretending this would be a regular lexibank dataset?

**Comment by LinguList** [comment]
Well, I'd say it is as much a lexibank dataset as any dialect data collection, and since it contains alignments, which I have already converted, it is worthwhile to provide it in the form of a CLDF dataset, even if there are no mappable concepts, but just strings that one can align and which have been aligned.

**Comment by LinguList** [comment]
I just didn't prioritize this dataset when finalizing the lexibank paper, as it is not really comparable to other datasets, similar to [holmie](https://github.com/digling/holmie).

**Comment by LinguList** [comment]
Please check how I did this now. If it should be released in any way, one would have to add the sources. I'd restrict those to the paper by Hruschka et al.

**Comment by LinguList** [comment]
And adding a note that says that this is a very artificial dataset in some sense is also useful.
